src/main/java/edu/csula/datascience/acquisition/ReadJsonTest.java

```diff
@@ -0,0 +1,185 @@
+package edu.csula.datascience.acquisition;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.mongodb.BulkWriteOperation;
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import com.mongodb.Mongo;
+import com.mongodb.MongoClient;
+import com.mongodb.MongoException;
+import com.mongodb.util.JSON;
+
+public class ReadJsonTest {

```

Comment:
This shouldn't really be called Test ... it's an application or some sort of application type. It's confusing naming.